### ISSUE TITLE
IKEA five button remote multiple issues after pairing

### ISSUE THREAD

**Issue by tubalainen** [dresden-elektronik/deconz-rest-plugin#3328]
## Describe the bug

Two issues in one - related?
1.) Just added remote controls (switches) does not work proper - several reboots of deconz and home assistant required (very odd).
2.) Long release, button hold does not work (x001, x003)

## Steps to reproduce the behavior

- Pairing of remote works as expected (adding a switch in Phoscon)
- (Renaming issue, see separate issue) https://github.com/dresden-elektronik/deconz-rest-plugin/issues/3332
- After restart of deconz - BUG - The right deconz_event is being sent to Home Assistant (in my case) but the automation does not trigger even if the "debug" clearly states the event data id is the new name I´ve set.
- There isnt any support for long press, long release and button hold. Just "short release" (x002) - Only x002 shows up in both home assistant and deconz logs.
- After several restarts of deconz and home assistant - it suddenly kicks to life and works, the deconz_event triggers and the automation just works - NOTHING changed, just restarts.
- BUG - The issue with Long press and long release remains. It just sends x002 never x001 and x003.

I have repeated this with three remotes (switches), IKEA on/off and IKEA 5 button round´s - all exactly the same behavior as above.

## Expected behavior

1.) No multiple restarts should be needed after adding a remote control.
2.) The multiple button functions should work, long release and button hold

## Environment

 - Host system: Debian 9 under ESXi VM
 - Running method: Marthoc Docker container
 - Firmware version: (26370500)
 - deCONZ version: (.82)
 - Device: ConBee I
 - Do you use an USB extension cable: no

**Comment by SAOPP** [comment]
https://github.com/dresden-elektronik/deconz-rest-plugin/issues/2618#issuecomment-649822696

**Comment by Kane610** [comment]
This looks like a mess of hass related and deconz related issues 😣

**Comment by Mimiix** [comment]
@tubalainen Thanks for the extensive bug report. Could you make a different bug report for the renaming stuff? Now there's 2 things messed up in 1 issue.

**Comment by Mimiix** [comment]
@tubalainen Ehh?

**Comment by tubalainen** [comment]
> @tubalainen Ehh?

Do not know what happened here, can you please reopen? Did not mean to close it. Must have been a slip of a button. My bad.

**Comment by Mimiix** [comment]
@tubalainen No problem 👍

**Comment by tubalainen** [comment]
Will split it soon as you suggested. :)

**Comment by tubalainen** [comment]
@Mimiix Here Dennis, the new renaming issue: https://github.com/dresden-elektronik/deconz-rest-plugin/issues/3332

**Comment by MarcoStein** [comment]
I have a similar problem with the IKEA 5 button remote, but specifically in conjunction with the left and right buttons of the tradfri remote. I cannot use them to manage the IKEA lamps white tone (colder .. warmer)

**Comment by tubalainen** [comment]
> I have a similar problem with the IKEA 5 button remote, but specifically in conjunction with the left and right buttons of the tradfri remote. I cannot use them to manage the IKEA lamps white tone (colder .. warmer)

I had a similar problem in my bed room. One remote to control two lights. Dont remember how I solved it. I used one of the options below. :)
1.) Try to add the lights in question to a Phoscon group with the remote control and it should work.
2.) If that does not work, first join the remote to Phoscon - Then join it to the light (being very close to the bulb and hold down the join button on the remote).
![image](https://user-images.githubusercontent.com/26795429/94453628-354cd900-01b1-11eb-8597-a8543a669675.png)
